modules/accounts/contacts.nix

```diff
@@ -126,6 +126,7 @@ in {
         contactOpts
         (import ../programs/vdirsyncer-accounts.nix)
         (import ../programs/khal-accounts.nix)
+        (import ../programs/khard-accounts.nix)
```

Comment:
Hmm, these imports seem like a mistake to me? I think putting something like

```
options.accounts.contact.accounts = mkOption {
  type = with types; attrsOf (submodule (import ./khard-accounts.nix));
};
```

inside the `khard.nix` file would be more idiomatic. For example, the various email related modules such as `notmuch.nix`.

Reply:
Btw, `notmuch.nix` doesn't seem to import any other file. There is a `notmuch-virtual-mailbox.nix` but it's imported in `neomutt.nix`. I don't use neomutt, notmuch or the mail infra on home-manager so I'm a bit confused about what's going on here or what you are referring to.

Reply:
Ok I've moved the contents of `khard-accounts.nix` into `khard.nix` now.